lib/router.js

```diff
@@ -45,18 +46,26 @@ router.get('/slashCommands/ci', (req, res) => {
 router.get('/slashCommands/ext', (req, res) => {
     const command = req.query && req.query.command;
     const text = req.query.text;
+    const update = req.query.update;
 
     if (command && command === '/ramal') {
-        const findExtension = text ?
-            extCommand.find.bind(extCommand, text) :
-            extCommand.find.bind(extCommand);
-
-        findExtension(out => {
-            res.send(out);
-        });
+        if (update && update === 'true' && config.extMode === 'SHEET') {
+            extCommand.update(text, out => {
```

Comment:
Regardless if we decide to let the user search && updated at the same time, we shouldn't hardcode that behaviour into the `update` method itself, that was my point. That's [Clean Coding](http://www.amazon.com.br/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882)... A method shouldn't do more than it says. It shouldn't have more than one clear responsibility, know what I mean? So even if we decide to do the `search + update` combo, it should be done with two separate calls to two separate, specific, singly-responsible methods. Do you agree with that approach?